pkg/statistics/table.go

```diff
@@ -72,6 +72,14 @@ type Table struct {
 	// 1. Initialized by snapshot when loading stats_meta.
 	// 2. Updated by the analysis time of a specific column or index when loading the histogram of the column or index.
 	LastAnalyzeVersion uint64
+	// LastStatsFullUpdateVersion is the mvcc version of the last full update of histograms.
+	// It differs from LastAnalyzeVersion because it can be influenced by some DDL.
+	// e.g. When we execute ALTER TABLE ADD COLUMN, there'll be new record inserted into mysql.stats_histograms.
+	//      We need to load the corresponding one into memory too.
+	// It's used to skip redundant loading of stats, i.e, if the cached stats is already update-to-date with mysql.stats_xxx tables,
+	// and the schema of the table does not change, we don't need to load the stats for this table again.
+	// Stats' sync load/async load should not change this field since they are not table-level update.
+	LastStatsFullUpdateVersion uint64
```

Comment:
Maybe we can make it LastHistogramUpdateVersion?  Not sure if `full` makes sense here.

Reply:
Yes, I think we can change it for a better understanding.